sktime/clustering/kmeans.py

```diff
@@ -401,15 +401,44 @@ class MiniBatchKmeansClustering(KmeansClustering):
     KmeansClustering : k-means clustering without mini-batching
     """
 
-    def __init__(self, n_clusters, max_iter=5, metric=None, tolerance=1e-5, init_strategy='kmeans++',
+    def __init__(self, n_clusters, batch_size=100, max_iter=5, metric=None, tolerance=1e-5, init_strategy='kmeans++',
                  n_jobs=None, initial_centers=None):
         """
-        Constructs a Minibatch k-means estimator. For a detailed argument description, see :class:`KmeansClustering`.
+        Constructs a Minibatch k-means estimator. For a more detailed argument description,
+        see :class:`KmeansClustering`.
+
+        The only additional parameter is batch_size, which determines the maximum sample size if calling
+        :meth:`fit()`.
         """
         super(MiniBatchKmeansClustering, self).__init__(n_clusters, max_iter, metric,
                                                         tolerance, init_strategy, False,
                                                         n_jobs=n_jobs,
                                                         initial_centers=initial_centers)
+        self.batch_size = batch_size
+
+    def fit(self, data, initial_centers=None, callback_init_centers=None, callback_loop=None, n_jobs=None):
```

Comment:
is this tested anywhere

Reply:
yes! the tests cover this piece actually, i was surprised too